webpack.config.js

```diff
@@ -0,0 +1,18 @@
+const path = require('path');
+
+module.exports = {
+  entry: {
+    'js/search/timetable': './js/timetable.js.jsx'
```

Comment:
I've moved the raw file out of `public/`. Eventually I'll do the same for the other files.